src/main/resources/com/microsoft/jenkins/azuread/AzureSecurityRealm/config.jelly

```diff
@@ -46,6 +46,9 @@
             <f:checkbox />
         </f:entry>
 
+        <f:entry title="${%Domain Hint: The realm of the user in a federated directory}" field="domainHint">
```

Comment:
the label would normally just be the field name and then either a description or help text to explain what its for